R/wrappers.R

```diff
@@ -119,20 +119,15 @@ saveWorkbook <- function(wb, file, overwrite = FALSE, returnValue = FALSE) {
 
   xlsx_file <- wb$saveWorkbook()
 
-  result<-tryCatch(file.copy(from = xlsx_file, to = file, overwrite = overwrite),
-    error = function(e) e, warning = function(w) w)
-
-
-
+  result <- file.copy(from = xlsx_file, to = file, overwrite = overwrite)
 
   ## delete temporary dir
   unlink(dirname(xlsx_file), force = TRUE, recursive = TRUE)
-  if(returnValue == FALSE){
+  if (returnValue == FALSE) {
```

Comment:
@ycphs 
Do you know if there any reason why we wouldn't want or be able to simply return the result of the `file.copy()`?  Does using `invisible(1)` satisfy anything?

Reply:
Initially there was no return value and  the feature request to have an optional made it necessary to have a default with no return value.

Reply:
The default value returned was `invisible(1)` which I don't see as useful as having the return value be the result of `file.copy()` instead -- especially since if `file.copy()` fails `saveWorkbook()` essentially fails.  So I guess I'm asking: should `invisible(1)` _ever_ be returned or should the result of  just be `file.copy()` (invisibly) returned?

Reply:
yes that would make more sense and it helps to actually and leads to the initial error.